app/models/geographic_area.rb

```diff
@@ -163,6 +163,17 @@ def self.parent
     end
   }
 
+  scope :with_data_origin, -> (data_origin) {
+    where(data_origin: data_origin) unless data_origin.blank?
```

Comment:
You can use the new Ruby pattern `where(data_origin:)` without reference to the value if there is an identically named variable, like in this case.